source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -122,19 +145,18 @@ EmphasisParser::EmphasisState EmphasisParser::MatchEmphasis(EmphasisParser &pars
         parser.ResetCurrentEmphasisState();
         parser.UpdateLookBehind(stream.peek());
         token += stream.get();
-
         return EmphasisState::Text;
     }
     return EmphasisState::Emphasis;
 }
 
 // Captures remaining charaters in given token
 // and causes the emphasis parsing to terminate
-void EmphasisParser::Flush(std::string& currentToken)
+void EmphasisParser::Flush(int ch, std::string& currentToken)
```

Comment:
>int [](start = 27, length = 3)

char

Reply:
this is intentional.

I need to be able to handle EOF from stringstream which is -1.

---
In reply to: [158142461](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158142461) [](ancestors = 158142461)